### PR TITLE
qiskit programming - aus

### PR DESCRIPTION
**Pull request by vabarbosa**
## Changes

add problem sets for Qiskit Programming Aus

## Implementation details

- added problem set to `toc.yaml`
- added problem set notebook

## How to read this PR

1. review problem set notebook
2. review changes in `toc.yaml`
3. test problem set question in preview (or local deployment)

[preview link](https://platypus-pr-1968.ecy0akwlcpw.us-south.codeengine.appdomain.cloud/problem-sets/qiskit_programming_australia/problem-set-1---beginners39-introduction)

### CODE REVIEW

**Comment by Astri-Cornish** [comment]
Hi @vabarbosa I just adding the grading meta for the other exercises. Not sure on ex1.3 and 2.2 though as there are multiple things to grade for those exercises - could you please check if I've done that correctly?

**Comment by vabarbosa** [comment]
> Hi @vabarbosa I just adding the grading meta for the other exercises. Not sure on ex1.3 and 2.2 though as there are multiple things to grade for those exercises - could you please check if I've done that correctly?

hi @Astri-Cornish for grading we only pass a single object there are a couple approaches for grading multiple objects which we can review

**Comment by vabarbosa** [comment]
i have [pushed updates for prob1-ex3, prob2-ex1, prob2-ex2](https://github.com/Qiskit/platypus/pull/1968/commits/0f81251b8be10a51da11b480c529ea393ef4b063) to workaround grading functions and multiple inputs

File: notebooks/problem-sets/qiskit_programming_australia.ipynb [hunk 1]
```diff
@@ -0,0 +1,490 @@
...
...o0mV9TDRIWxx3npcRxbnsop75XnVNH+XVTbnJZpyvMmQetkFuDe8I4yah6luJqr+Tny5JXAHePNUpkxMlzm31KPq7rQd76nuom5mnz+DU31dQjZUg73ylD2nc4BHAYdud4Vk65Om6KoA++l+VHfz72S/+V57iemDD649Ix2kUL/s5YYsHFl7WhIurWP1Q7GO+Me4I7M+9B12o8DBTtmR5F+XyohI2deij021VjHjJWegfSzBNJ7fNa87ZOffoqnHV0APkBG+VK5lzdp4sjuRthOvO0lDmnTNm6Mc53JaLN/uyalLAVbRog+++53v1t961vfchWf+MQnqh/5kR+pPvaxj3kSMqneAP/vfe971T/8wz/4WyosEfr4xz9e/cIv/EJ17733Tsr+gb63Ae4Tuhelx7aQeN7ZFnKXBr0fMUuQEhpxniDYKEzWeDeej1qJ8uLmG6/rJUCVDWDHs8NMNN7oSYJp4yQirIBCzAyx6RbNzg/tHco796APIaJOT1xqpT9OObTJoN1MWFBKrHdnqRHt9Q4zAvEAqXUM9BvGD4/7E3o5lfbeX3vc+yjidWwT6OrsSb0I9Z4mkK+9Ux09/IbXtFcvvlvtePek1rprKDSDutUCxohA+84rNXllj/YPaYkXH1ZSvEugfcfeXc14ythJTEk+r2PGUjMGc68ej1vuMTbHhPRBk6KVlv7iiPwZuKtPkassFekquhzXOU/clT7XutLkWuKk7YqZID788MPm0wMPPDBxgpi2w6t2aO7RXwq037qn1kHQ0nw3opU5fcLli0se9aHv1fW6Lp2Ttkzf0EDiImwBXDV90MP18Ci/0UlbrhXpc68sjzppq+uu0xZVTz2NvPMyOsCdScAl+U4L/aGDpyw4TfwFTrU+/IF3LJFBDjjWVf92dVjGWNnvXenGXYMHzTEuUcd18swbGMOT8kc+sZE8ieeJD+9XcD1YgD6zp1wYgXfibIcLHNKmzTLUUS/Xsb0HNSHlaT8gnkncMsLL2knvO9/5TvW1r32t+sd//EdjiM985jMVx8/+7M9Whw4dmlht3i3793//9+qv/uqvqteFlQDtn/70p6vPf/7zFTrycg0b4D6h5zMDZneZszpQFm1PVQQyYt2lUErPB+cID9svUX5eguKREgGvex4lXcFMWGnHhloJcT/1j01b3CCt6SZ/cb19ShqONnD3ciEpD
```
Comment (vabarbosa) [inline notebooks/problem-sets/qiskit_programming_australia.ipynb:318]: @Astri-Cornish FYI, `to_numpy_matrix` has been removed in networkx 3: 

see https://networkx.org/documentation/stable/release/migration_guide_from_2.x_to_3.0.html#replacing-numpy-scipy-matrices-with-arrays

got this error when testing:
image

Comment (Astri-Cornish) [reply notebooks/problem-sets/qiskit_programming_australia.ipynb]: Thanks for this! I have just added a commit to fix this here and in the grader

**Pull request merged**